saleor/graphql/channel/types.py

```diff
@@ -160,7 +165,15 @@ class Meta:
         )
 
 
-class Channel(ModelObjectType[models.Channel]):
+class OrderSettings(ObjectType):
+    automatically_confirm_all_new_orders = graphene.Boolean(required=True)
+    automatically_fulfill_non_shippable_gift_card = graphene.Boolean(required=True)
```

Comment:
We should also provide descriptions here, it can be the same if we have it somewhere else (I think we have in inputs).